src/components/questions/modals/AnswerModal.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+
+import { makeStyles } from '@material-ui/core/styles';
+import Modal from '@material-ui/core/Modal';
+import Backdrop from '@material-ui/core/Backdrop';
+import Fade from '@material-ui/core/Fade';
+
+
+import AnswerForm from './AnswerForm.js';
+
+const useStyles = makeStyles(theme => ({
+    modal: {
+        display: 'flex',
+        alignItems: 'center',
+        justifyContent: 'center',
+
+    },
+    paper: {
+        backgroundColor: theme.palette.background.paper,
+        border: '2px solid #000',
+        boxShadow: theme.shadows[5],
+        padding: theme.spacing(4, 10, 6),
+        // padding: theme.spacing(2, 4, 3),
+    },
+}));
+
+var AnswerModal = (props) => {
+    const classes = useStyles();
+    const [open, setOpen] = React.useState(false);
+
+    const handleOpen = () => {
+        setOpen(true);
+    };
+
+    const handleClose = () => {
+        setOpen(false);
+    };
+
+    const mystyle = {
+      backgroundColor: "white",
+      padding: "0px",
+      fontSize: "small",
+      border: 'none',
+      cursor: 'pointer',
+    }
+
+    return (
+        <div>
+
+            <button style = {mystyle} onClick={handleOpen}>
+                Add Answer
+            </button>
```

Comment:
is there a specific reason you're using `<button>` here rather than the Material-UI `<Button>` component?